bin/mujin_webstackclientpy_downloaddata.py

```diff
@@ -75,6 +75,7 @@ def _DownloadBackup(webClient, sceneList, timeout=600.0):
     with tarfile.open(fileobj=response.raw, mode='r|gz') as tar:
         tar.extractall(path=downloadDirectory)
     log.info('download completed, data saved to: %s', downloadDirectory)
+    print downloadDirectory # other scripts can read stdout and learn the directory path
```

Comment:
@LLipter webstack client is supposed to be python3 compatible, please use `print(downloadDirectory)` instead. 

Also, I think we should print this information in the main function, not inside the `_DownloadBackup` function since relaying the download directory to another script is not `_DownloadBackup`s concern.